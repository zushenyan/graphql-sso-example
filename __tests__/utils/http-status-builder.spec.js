const {
  buildMessage,
  createInternalError,
  createOk
} = require("utils/http-status-builder.js");

describe("", () => {
  describe("buildMessage", () => {
    it("should work", () => {
      const result = {
        status:  456,
        message: "lalal"
      };
      expect(buildMessage(result.status, result.message)).toEqual(result);
    });
  });

  describe("createInternalError", () => {
    it("should work", () => {
      expect(createInternalError()).toMatchSnapshot();
    });
  });

  describe("createOk", () => {
    it("should work", () => {
      expect(createOk(123)).toMatchSnapshot();
    });
  });
});
