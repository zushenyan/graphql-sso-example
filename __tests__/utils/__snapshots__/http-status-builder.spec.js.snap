// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` createInternalError should work 1`] = `
Object {
  "message": "internal server error",
  "status": 500,
}
`;

exports[` createOk should work 1`] = `
Object {
  "message": "ok",
  "status": 123,
}
`;
